### PR TITLE
Setting executable bits on three files in sandbox Fix/798

### PR DESCRIPTION
**Pull request by alameldin**
“fixes issue #798 ”

### CODE REVIEW

**Comment by ctb** [comment]
ok to test

**Comment by alameldin** [comment]
- [ ] Is it mergeable?
- [ ] Did it pass the tests?
- [ ] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [ ] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [ ] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [ ] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [ ] Was a spellchecker run on the source code and documentation after
  changes were made?

**Comment by ctb** [comment]
Looks good; merging.  Thank you!

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,7 @@
+2015-02-21  Hussien Alameldin  <hussien@msu.edu>
+
+   * sandbox/calc-best-assembly.py, collect-variants.py, graph-size.py: Set executable bits using "chmod +x"
```
Comment (mr-c) [inline ChangeLog]: This line is over 80 characters long. @ctb

Comment (alameldin) [inline ChangeLog]: should i edit it?

**Pull request merged**